martin-mbtiles/src/tile_copier.rs

```diff
@@ -70,13 +76,7 @@ impl TileCopier {
         })
     }
 
-    pub async fn run(self) -> MbtResult<()> {
-        let opt = SqliteConnectOptions::new()
```

Comment:
why remove it? We should still validate that we are not doing a copy operation on a random sqlite file

Reply:
Right, if we `force-simple` we don't use the `storage_type` which is why I moved it but it's still valuable to verify that it is a valid `storage_type`. Will add it back